docs/docs/deploying-to-s3-cloudfront.md

```diff
@@ -54,6 +54,23 @@ And finally, add the deployment script to your `package.json`:
 That's it!
 Run `npm run build && npm run deploy` to do a build and have it immediately deployed to S3!
 
+## Deploying with `.env` variables
+
+Some deployments require passing environment variables. To deploy with environment variables, update the deployment script to your `package.json`:
+
+```js
```

Comment:
We can add the filename to the code block for an extra tip:
```suggestion
```js:title=package.json
```